Jamfile

```diff
@@ -213,6 +213,8 @@ rule linking ( properties * )
 		result += <library>/try_signal//try_signal/<link>static ;
 	}
 
+	result += <linkflags>"-lxnvme -laio -luuid -lnuma -pthread -lrt" ;
```

Comment:
What about android/windows/darwin? Will it build there?

Reply:
We are currently working on Windows support for xNVMe and expect a release in couple of months. It'll be using I/O Control Ports for the async IO.

xNVMe is currently untested on Darwin, but does run on FreeBSD. Android is also untested, but at least the POSIX interfaces should work, and then depending on the kernel it should support libaio and io_uring. I expect that we'll get test coverage for both Darwin and Android :)